inc/network/includes-network.php

```diff
@@ -0,0 +1,735 @@
+				</td>
+			</tr>
+		</table>
+		<?php submit_button( __( 'Install' ), 'primary', 'submit' ); ?>
+	</form>
+	<?php
+}
+
+/**
+ * Prints step 2 for Network installation process.
+ *
+ * @since 3.0.0
+ *
+ * @global wpdb $wpdb     WordPress database abstraction object.
+ * @global bool $is_nginx Whether the server software is Nginx or something else.
+ *
+ * @param false|WP_Error $errors Optional. Error object. Default false.
+ */
+function network_step2( $errors = false ) {
+	global $wpdb, $is_nginx;
+
+	$hostname          = get_clean_basedomain();
+	$slashed_home      = trailingslashit( get_option( 'home' ) );
+	$base              = parse_url( $slashed_home, PHP_URL_PATH );
+	$document_root_fix = str_replace( '\\', '/', realpath( $_SERVER['DOCUMENT_ROOT'] ) );
+	$abspath_fix       = str_replace( '\\', '/', ABSPATH );
+	$home_path         = 0 === strpos( $abspath_fix, $document_root_fix ) ? $document_root_fix . $base : get_home_path();
+	$wp_siteurl_subdir = preg_replace( '#^' . preg_quote( $home_path, '#' ) . '#', '', $abspath_fix );
+	$rewrite_base      = ! empty( $wp_siteurl_subdir ) ? ltrim( trailingslashit( $wp_siteurl_subdir ), '/' ) : '';
+
+	$location_of_wp_config = $abspath_fix;
+	if ( ! file_exists( ABSPATH . 'wp-config.php' ) && file_exists( dirname( ABSPATH ) . '/wp-config.php' ) ) {
+		$location_of_wp_config = dirname( $abspath_fix );
+	}
+	$location_of_wp_config = trailingslashit( $location_of_wp_config );
+
+	// Wildcard DNS message.
+	if ( is_wp_error( $errors ) ) {
+		echo '<div class="error">' . $errors->get_error_message() . '</div>';
+	}
+
+	if ( $_POST ) {
+		if ( allow_subdomain_install() ) {
+			$subdomain_install = allow_subdirectory_install() ? ! empty( $_POST['subdomain_install'] ) : true;
+		} else {
+			$subdomain_install = false;
+		}
+	} else {
+		if ( is_multisite() ) {
+			$subdomain_install = is_subdomain_install();
+			?>
+	<p><?php _e( 'The original configuration steps are shown here for reference.' ); ?></p>
+			<?php
+		} else {
+			$subdomain_install = (bool) $wpdb->get_var( "SELECT meta_value FROM $wpdb->sitemeta WHERE site_id = 1 AND meta_key = 'subdomain_install'" );
+			?>
+	<div class="error"><p><strong><?php _e( 'Warning:' ); ?></strong> <?php _e( 'An existing WordPress network was detected.' ); ?></p></div>
+	<p><?php _e( 'Please complete the configuration steps. To create a new network, you will need to empty or remove the network database tables.' ); ?></p>
+			<?php
+		}
+	}
+
+	$subdir_match          = $subdomain_install ? '' : '([_0-9a-zA-Z-]+/)?';
+	$subdir_replacement_01 = $subdomain_install ? '' : '$1';
+	$subdir_replacement_12 = $subdomain_install ? '$1' : '$2';
+
+	if ( $_POST || ! is_multisite() ) {
+		?>
+		<h3><?php esc_html_e( 'Enabling the Network' ); ?></h3>
+		<p><?php _e( 'Complete the following steps to enable the features for creating a network of sites.' ); ?></p>
+		<div class="notice notice-warning inline"><p>
+		<?php
+		if ( file_exists( $home_path . '.htaccess' ) ) {
```

Comment:
This condition can probably be removed since there will not be an `.htaccess`

Reply:
It's not _necessary_ but we don't need to do it. We'll never have an .htaccess file to check if it exists, so our code will never stop here. I think if we're going to make improvements to the page, we might as well make it better than it was before.

Reply:
Sounds good. Done!